Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -263,6 +263,8 @@ final class ConfigGenerator: ConfigGenerating {
             }
         case let .macOS(version):
             settings["MACOSX_DEPLOYMENT_TARGET"] = .string(version)
+        case let .watchOS(version):
+            settings["WATCHOS_DEPLOYMENT_TARGET"] = .string(version)
```

Comment:
I'd add a test case for this, similarly to `MACOSX_DEPLOYMENT_TARGET`

Reply:
Added